theme/theme.go

```diff
@@ -321,6 +327,11 @@ func Padding() float32 {
 	return current().Size(SizeNamePadding)
 }
 
+// ParagraphPadding is the standard gap between paragraphs with rich text.
+func ParagraphPadding() float32 {
+	return current().Size(SizeNameParagraphPadding)
+}
+
```

Comment:
This helper probably isn't needed as Paragraph Padding is only used in two places.

Reply:
These are also a public helper for people writing their own widgets... so maybe it is useful...
It should, however have a Since line.